rpcwebsocket.go

```diff
@@ -36,8 +36,19 @@ const (
 	// handler since notifications have their own queuing mechanism
 	// independent of the send channel buffer.
 	websocketSendBufferSize = 50
+
+	maxConcurrentWebsocketClientRequests = 20
 )
 
+type semaphore chan struct{}
+
+func makeSemaphore(n int) semaphore {
+	return make(chan struct{}, n)
+}
+
+func (s semaphore) p() { s <- struct{}{} }
```

Comment:
I think these should be named `Acquire` and `Release`.  I'm of course familiar with the `P(S)` and `V(S)` syntax when dealing with semaphores, but I prefer the code to be readable by everyone and that nomenclature is not particularly illuminating for those who aren't familiar with primitive semaphores.